src/org/opensolaris/opengrok/history/HistoryGuru.java

```diff
@@ -341,69 +343,95 @@ public void addRepositories(File[] files, Collection<RepositoryInfo> repos,
      */
     private void addRepositories(File[] files, Collection<RepositoryInfo> repos,
             IgnoredNames ignoredNames, boolean recursiveSearch, int depth) {
+
+        // This check for ignoring a repository only works for embedded
```

Comment:
OK, let me test my understanding. You have a pull request with the suggested changes. Do you want me to copy the code out of that and apply it to my forked repository along with adding the Unit Test you created, then promote it back into my repository so that it gets associated with my original pull request?

You sent me some other mail talking about the secret file not being a good mechanism and it should be taken care of via Configuration mechanism. Is that a longer term situation and these changes just a band-aid for now?